src/main/java/org/ohdsi/standardized_analysis_api/cohortcharacterization/design/CriteriaFeature.java

```diff
@@ -0,0 +1,15 @@
+package org.ohdsi.standardized_analysis_api.cohortcharacterization.design;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import org.ohdsi.circe.cohortdefinition.CriteriaGroup;
+
+@JsonPropertyOrder({ "name", "criteriaGroup" })
+public interface CriteriaFeature {
+
+    @JsonGetter("name")
+    String getName();
+
+    @JsonGetter("criteriaGroup")
```

Comment:
Let's call this field 'expression'. It's a bit awkward to refer to the field as the underlying class.